tests/conftest.py

```diff
@@ -41,7 +41,7 @@ def mask():
 
 @pytest.fixture
 def bboxes():
-    return [[15, 12, 75, 30, 1], [55, 25, 90, 90, 2]]
+    return [[0.15, 0.12, 0.75, 0.30, 1], [0.55, 0.25, 0.90, 0.90, 2]]
```

Comment:
why did you changed this?

Reply:
Interger bboxes throw errors while checking **test_serialization.py**.
RandomSizedBBoxSafeCrop only allows float bboxes now.
In my understanding, float bboxes is standard for this library, so I changed it.
Am I wrong?

Reply:
@albu He is right, tests functions does not call Compose transform, so bboxes  in test are not normalized. 

I think, need rename this function to `normalized_bboxes` and return back original function. Then in `test_serialization` rename all `bboxes` test functions params to `normalized_bboxes`, except `test_transform_pipeline_serialization_with_bboxes`

Reply:
or `albumentations_boxes`.

Reply:
@Dipet Fixed.